mailpoet/lib/Services/AuthorizedSenderDomainController.php

```diff
@@ -46,14 +46,20 @@ public function getDomainRecords(string $domain = ''): array {
    *
    * Note: This includes both verified and unverified domains
    */
-  public function getAllSenderDomains(): array {
+  public function getAllSenderDomains(bool $skipCache = false): array {
```

Comment:
I find this type of boolean parameter not very readable. I would prefer creating a new public method that would say what it does in its name.

`$controller->getAllSenderDomains(true)`
vs
`$controller->getAllSenderDomainsIgnoringCache()`

What do you think?

Reply:
Fixed in https://github.com/mailpoet/mailpoet/pull/4363/commits/236b75eddf091510f9beaf12e3923629cced2ace